packages/cli/test/unit/commands/help.test.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  calcLineLength,
+  help,
+  lineToString,
+  outputArrayToString,
+} from '../../../src/commands/help';
+import { deployCommand } from '../../../src/commands/deploy/command';
+
+import chalk from 'chalk';
+
+describe('help command', () => {
+  describe('calcLineLength', () => {
+    test.each([
+      {
+        name: 'without ansi',
+        line: ['a line without ansi'],
+        expectedLength: 19,
+      },
+      {
+        name: 'with ansi',
+        line: [`a line with ${chalk.red('ansi')}`],
+        expectedLength: 16,
+      },
+    ])(
+      'should calculate the correct line length $name',
+      ({ line, expectedLength }) => {
+        expect(calcLineLength(line)).toBe(expectedLength);
+      }
+    );
+  });
+
+  describe('lineToString', () => {
+    test.each([
+      {
+        line: ['a', 'b', 'c'],
+        expected: 'a b c',
+      },
+      {
+        line: [' ', 'a', ' ', 'b', ' ', 'c', ' '],
+        expected: ' a b c ',
+      },
+      {
+        line: [' ', '  ', '   '],
+        expected: '      ',
+      },
+      {
+        line: ['a', '  ', '   ', 'b', 'c'],
+        expected: 'a     b c',
+      },
+    ])(
+      'should insert spaces between non-whitespace items only; $line',
+      ({ line, expected }) => {
+        expect(lineToString(line)).toBe(expected);
+      }
+    );
+  });
+
+  describe('outputArrayToString', () => {
+    test('should join a list of strings using newlines', () => {
+      expect(outputArrayToString(['line 1', 'line 2', 'line 3'])).toBe(
+        'line 1\nline 2\nline 3'
+      );
+    });
+  });
+
+  test('help', () => {
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏add a small column test

Can we add another test with a smaller column size to exercise word wrapping?

Reply:
I can't figure out how to get jest to respect changes to `process.stdout.columns` :(

Reply:
aye i figured it out. fixes pushing now